src/guide/jbs-jdk-bug-system.md

```diff
@@ -122,11 +122,11 @@ Add a comment when adding a (Rel)[-wnf]{.jbs-label} label so that it's clear for
 
 ![Guidelines for setting [Affects Version/s]{.jbs-field}](affects_versions.svg)
 
-1. Issue relevant to JDK 8 and all future releases.
-2. No need to add additional releases as they are implied.
+1. Affects Version contains 8 only, which means this issue is applicable to 8 and all future releases (until it's fixed).
```

Comment:
```suggestion
1. [Affects Version/s]{.jbs-field} contains 8 only, which means this issue is applicable to 8 and all future releases (until it's fixed).
```
Shall we mark-up the JBS field?

Reply:
Right. I meant to do that but pushed in a hurry when I left the office... 😅